### ISSUE TITLE
Add support for using a pre-existing feature.xml

### ISSUE THREAD

**Issue by jvz** [lburgazzoli/gradle-karaf-plugin#2]
There are two facets to this:

1. Include or merge together (if possible) `src/main/feature/feature.xml` (configurable location?) with the inline feature XML slurper thing into a kar file.
2. Allow the feature.xml file to be interpolated with gradle properties like how karaf-maven-plugin works. The standard interpolation was `${foo}` and `@foo@`.

**Comment by jvz** [comment]
By the way, I've since migrated to doing it all in-gradle, but this feature is still useful for users migrating from the maven karaf plugin.

**Comment by lburgazzoli** [comment]
Yep, I hope to have time to implement it once main functionalities are stable and documented

**Comment by tonit** [comment]
Hi, you may prioritize this again since even karaf core developers tend to hand-roll feature.xmls these days. The only thing for your plugin would be to support extend the dsl to support this either by configuration or convention (say, pick up the feature.xml at a known location). You may also include a way to disable merging (as discussed in this item). All i really would need is a way to tell the plugin here is the xml, pls attach it to the gradle results and maybe use it when baking a kar file (though this is not needed for me..)
